pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java

```diff
@@ -178,7 +180,7 @@ private void initPendingAckStore() {
                     recoverTime.setRecoverStartTime(System.currentTimeMillis());
                     pendingAckStore.replayAsync(this, internalPinnedExecutor);
                 }).exceptionally(e -> {
-                    acceptQueue.clear();
+                    handleCacheRequest();
                     changeToErrorState();
```

Comment:
Two questions:

1. After the state of `PendingAckHandle` change to `Error`, will some tasks still will append to the `acceptQueue`? If yes, we should prevent that continue to append any task to the queue after the stat changed to `Error`
2. Suggest that just change the future in the queue directly to exceptionally complete because `pendingAckStoreFuture` has failed then no more tasks can be executed successfully.

Reply:
change it `exceptionallyAsync` use internalPinnedExecutor can better fix this case